src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -116,7 +116,6 @@ const Body = BaseForm.extend({
   doCustomURI () {
     return request({
       url: this.customURI,
-      method: 'POST',
```

Comment:
shall we add `method` to the API response in order to make UI dumb ?
https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/idp/idx/data/identify-with-device-launch-authenticator.json#L70

Reply:
Yea, that's mentioned in some discussion. But no decision yet. I can follow up later.

Reply:
could you cut a JIRA and add to the code as comment?

Reply:
https://oktainc.atlassian.net/browse/OKTA-278014